example/Vagrantfile

```diff
@@ -1,11 +1,17 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-docker-compose")
+        system('vagrant plugin install vagrant-docker-compose')
+        raise("Plugin installed. Run command again.");
+end
+
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/trusty64"
 
   config.vm.network(:forwarded_port, guest: 8080, host: 8080)
 
+  config.vm.provision :shell, inline: "sudo apt-get update"
```

Comment:
The `sudo` here is unnecessary, as the `:shell` provisioner has an option `:privileged` that defaults to `true`. Can you remove it?